packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -17,6 +21,10 @@ const FlexSpaceBetween = styled(FlexSpaceBetweenCenter)`
   align-items: flex-start;
 `;
 
+const SkeletonHeader = styled(FlexCenter)`
```

Comment:
You shouldn't need a custom component to add padding to this. FlexCenter extends [Material UI Box](https://v4.mui.com/components/box/#box) component and it is possible to use that to set the margin.
eg `<FlexCenter mt="10px">`

Reply:
Nice, will remove `SkeletonHeader`.